.github/workflows/android_build.yml

```
@@ -0,0 +1,24 @@
+name: Android Build
+
+on: pull_request
```

This is what I want, but I want to specify branch :) So, would you update like below?

```suggestion
on: pull_request
  branches: [ master ]
```